<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLCDNumber" name="NumberClient">
            <property name="digitCount">
             <number>5</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="intValue" stdset="0">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelClient">
            <property name="text">
             <string>Clients</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textClientList"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLCDNumber" name="NumberChannel">
            <property name="digitCount">
             <number>5</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="intValue" stdset="0">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelChannel">
            <property name="text">
             <string>Channels</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textChannelList"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="logText"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
